frontend/src/metabase/query_builder/components/QueryHeader.jsx

```diff
@@ -431,6 +439,28 @@ export default class QueryHeader extends Component {
             </Tooltip>
         ]);
 
+        if (!isNew && card.can_write && question.alertType() !== null) {
+            const createAlertItem = {
+                title: t`Get alerts about this`,
+                icon: "alert",
+                action: () => this.setState({ modal: "create-alert" })
+            }
+            const updateAlertItem = {
+                title: t`Alerts are on`,
+                icon: "alert",
+                content: (toggleMenu, setMenuFreeze) => <AlertListPopoverContent onCancel={toggleMenu} setMenuFreeze={setMenuFreeze} />
```

Comment:
Might be worth formatting this a little more nicely to make it a bit easier to read.